mlflow/models/model.py

```diff
@@ -91,6 +94,7 @@ def __init__(
         signature=None,  # ModelSignature
         saved_input_example_info: Dict[str, Any] = None,
         model_uuid: Union[str, Callable, None] = lambda: uuid.uuid4().hex,
+        mlflow_version: Union[str, Callable, None] = lambda: mlflow.version.VERSION,
```

Comment:
No need for a callable here since the version doesn't need to be dynamically generated (unlike UUID).

```suggestion
        mlflow_version: Union[str, None] = lambda: mlflow.version.VERSION,
```

Reply:
Will change